server/portal/settings/settings_custom.example.py

```diff
@@ -258,6 +258,20 @@
     "hideApps": False,
     "hideDataFiles": False,
     "hideAllocations": False,
+    "hideManageAccount": False,
     "onboardingCompleteRedirect": '/workbench/',
-    "noPHISystem": ""
+    "noPHISystem": "",
+    "customDashboardSection": {
+        "header": "My Account",
+        "links": [
+            {
+                "href": 'https://utexas.edu',
+                "text": 'Update Profile and Email Address',
+            },
+            {
+                "href": 'https://utexas.edu',
+                "text": 'Change Password',
+            }
+        ]
```

Comment:
**Request**: Make the links relative (e.g. `profile`, `change_password`) and use `urls.py` to capture and redirect the URLs to the correct domains and project directory.

**Benefit**: CMS Getting Started guide does not need URL management that only Portal-based CMS's will use.

**Illustration**: A TAM UR, like https://accounts-dev.tacc.utexas.edu/apcd/forgot_password, has variable domain (`accounts-**dev**.tacc.utexas.edu`), project directory (`/**apcd**/`), and base name (`**forgot_password**`).

Reply:
_**Note to Self**: If this can't come from Portal short-term; then add a redirect, for APCD, in the CMS, to the account creation URL._

Reply:
I added redirects from `/tam/register` to https://accounts-dev.tacc.utexas.edu/apcd/register on APCD Prod and QA servers.

Reply:
Can be implemented separately:
> [FP-1803](https://jira.tacc.utexas.edu/browse/FP-1803): TAM URLs in Portal that CMS Can Use